requests/database/data.graphql

```diff
@@ -1,92 +1,113 @@
 query {
-  data(
-    id: "810e84b4-9ebf-416c-88ea-aade848f1fdf"
-    timestamp: "2019-09-01T12:00:00+01:00"
-    locale: "en-GB"
-  ) {
-    uuid
-    timestamp
-    componentId
+  data(id: "7fc8c70b-24c2-4363-a05e-4ac39d38207f", locale: "en-US") {
```

Comment:
This query should be removed because it does not exist in `database.graphql`, that is, in the GraphQL specification that product-data databases need to adhere to.